app/client/components/consent/OnOffRadio.tsx

```diff
@@ -129,4 +128,16 @@ export class OnOffRadio extends Component<Props, {}> {
       </div>
     );
   }
+
+  public shouldComponentUpdate(nextProps: Props): boolean {
+    if (this.props.selectedValue === nextProps.selectedValue) {
+      return false;
+    }
+    return true;
```

Comment:
could simplify to 
`return this.props.selectedValue !== nextProps.selectedValue`

Reply:
you could make it even cleaner and turn it into an arrow function (which can be an expression and thus avoid the `return` keyword)
```suggestion
    public shouldComponentUpdate = (nextProps: Props) => this.props.selectedValue !== nextProps.selectedValue;
```

Reply:
👍